doc/long_tutorials/handle_drift/sg_execution_times.rst

```diff
@@ -0,0 +1,37 @@
+
+:orphan:
```

Comment:
this should be captured by the gitignore, but I think it should be `*sg_execution_times.rst` (instead of `doc/sg_execution_times.rst`)

@JoeZiminski

Reply:
Great thanks good point, have added here but opened a PR on main #3097